packages/ckeditor5-theme-lark/theme/ckeditor5-widget/widget.css

```diff
@@ -46,11 +46,19 @@
 	These styles show a different border for a blink of an eye, so `:focus` need to have same styles applied. */
 	&.ck-editor__nested-editable_focused,
 	&:focus {
-		@mixin ck-focus-ring;
 		@mixin ck-box-shadow var(--ck-inner-shadow);
 		@mixin ck-media-default-colors {
 			background-color: var(--ck-color-widget-editable-focus-background);
 		}
+
+		/*
+		* Fixes issue with the hidden border of the last focused cell during selection.
+		*
+		* See more: https://github.com/ckeditor/ckeditor5/issues/16979
```

Comment:
AFAIK, this fixes blue borders appearing in the bottom and right corners.

Reply:
Yep, comment needs to be adjusted